src/pages-and-resources/index.scss

```diff
@@ -1,3 +1,5 @@
+@import "./discussions/DiscussionAppList.scss";
```

Comment:
Sass-ignorant question: Why does this one get imported with a `.scss` suffix, while others have no suffix at all?

Reply:
Oops!  Cause I forgot to remove it.  Good catch. Adding the suffix is unnecessary, so I was going around removing it from them. :)